gravatar-ui/src/main/java/com/gravatar/ui/components/Profile.kt

```diff
@@ -84,8 +84,10 @@ public fun Profile(
                             UserInfo(state)
                         }
                     }
-                    Spacer(modifier = Modifier.height(16.dp))
-                    AboutMe(state)
+                    if (state.isAboutMeVisible()) {
+                        Spacer(modifier = Modifier.height(16.dp))
+                        AboutMe(state)
+                    }
```

Comment:
Originally, I wanted to have this check inside `AboutMe` component but having the `Spacer` made that impossible. Also, there's different spacing in the `LargeComponent` that didn't help.